setup.py

```diff
@@ -0,0 +1,30 @@
+from typing import List
+
+import setuptools
+
+__version__ = '2022.01.1'
```

Comment:
Not sure of a better way to handle this? having this in a single place would def be better + I don't think the leading `0` on the month good to have. I get this

```
UserWarning: Normalizing '2022.01.1' to '2022.1.1'
  warnings.warn(tmpl.format(**locals()))
```

Reply:
want to keep it as is, decode-config version numbers are in sync with tasmota since last year and Theo switched to calver using month with leading 0

Reply:
Loaded version from the script - one less place to maintain.